hack/shoot_gcp.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: core.gardener.cloud/v1beta1
 kind: Shoot
 metadata:
   name: ${GARDENER_CLUSTER_NAME}
+  annotations:
+    dashboard.garden.sapcloud.io/no-hibernation-schedule: 'true'
```

Comment:
If a cluster cannot get deprovisioned and just stays. Then it will be good if it hibernates till someone realizes it and will manually delete it. -> Are you sure to disable it?